launch/docker.go

```diff
@@ -74,6 +75,7 @@ func newDocker(setupImage, setupImageVer string, useSudo bool, interactiveMode b
 		localVolumes:      localVolumes,
 		buildUser:         buildUser,
 		dind: DinD{
+			enable:          dindEnable,
```

Comment:
The addition here will likely change the reference in the following line to d.dind.enable.

https://github.com/screwdriver-cd/sd-local/pull/91/files#diff-4f249559a0063721b9902972b3edda755ed28a3110c463e1de09ddb8bd87ecedR112

Reply:
Good!